moler/cmd/commandtextualgeneric.py

```diff
@@ -38,6 +38,9 @@ def __init__(self, connection, prompt=None, newline_chars=None, runner=None):
         self._last_not_full_line = None  # Part of line
         self._re_prompt = CommandTextualGeneric._calculate_prompt(prompt)  # Expected prompt on device
         self._newline_chars = newline_chars  # New line characters on device
+        self.do_not_process_after_done = True  # Set True if you want to break processing data when command is done. If
+        # False then on_new_line will be called after done if more lines are in the same data package.
```

Comment:
Do we really need it? It is default and as I can see below after `self.on_new_line()` if observer is done we don't process following lines. Moreover, while developing asyncio I've made fixes inside threaded runner that ensure `data_received()` won't be called for done observer - see: https://github.com/nokia/moler/blob/173ad0ad890882948c94650e328f424acaa7e218/moler/runner.py#L309

I know, inside previously send data there might be multiple lines but it is generic, phylosophical question "should we allow to process any data after observer is done"?
My understanding is NO.